src/acat/azext_acat/custom.py

```diff
@@ -0,0 +1,537 @@
+
+    def _execute_operations(self):
+        self.pre_operations()
+        self.CreateAcatReportWebhookWithDupAadToken(ctx=self.ctx)()
+        self.post_operations()
+
+    @classmethod
+    def _build_arguments_schema(cls, *args, **kwargs):
+        args_schema = super()._build_arguments_schema(*args, **kwargs)
+        from azure.cli.core.aaz import AAZListArg, AAZStrArg, AAZStrArgFormat
+
+        args_schema.content_type._required = False
+        args_schema.content_type._registered = False
+        args_schema.events._required = False
+        args_schema.events._registered = False
+        args_schema.payload_url._required = False
+        args_schema.payload_url._registered = False
+        args_schema.send_all_events._required = False
+        args_schema.send_all_events._registered = False
+        args_schema.status._required = False
+        args_schema.status._registered = False
+        args_schema.update_webhook_key._required = False
+        args_schema.update_webhook_key._registered = False
+        args_schema.webhook_key._required = False
+        args_schema.webhook_key._registered = False
+
+        args_schema.content_type_with_default = AAZStrArg(
+            options=["--content-type"],
+            arg_group="Properties",
+            help="content type",
+            required=False,
+            enum={"application/json": "application/json"},
+            default="application/json",
+        )
+        args_schema.events_with_default = AAZListArg(
+            options=["--events"],
+            arg_group="Properties",
+            help="under which event notification should be sent.",
+            required=False,
+            default=[],
+        )
+        args_schema.events_with_default.Element = AAZStrArg()
+        args_schema.payload_url_required = AAZStrArg(
+            options=["--payload-url"],
+            arg_group="Properties",
+            help="webhook payload url",
+            required=True,
+            fmt=AAZStrArgFormat(
+                pattern=r"^(http(s)?://)[\S]{0,64994}$",
+            ),
+        )
+        args_schema.trigger_mode = AAZStrArg(
+            options=["--trigger-mode"],
+            arg_group="Properties",
+            help="whether to send notification under any event.",
+            default="all",
+            enum={"all": "true", "customize": "false"},
+        )
+
+        args_schema.status_with_default = AAZStrArg(
+            options=["--disable"],
+            arg_group="Properties",
+            help="Webhook status.",
+            enum={"false": "enable", "true": "disable"},
+            default="enalbe",
+            blank="disable",
+        )
+        args_schema.webhook_key_with_default = AAZStrArg(
+            options=["--secret"],
+            arg_group="Properties",
+            help="webhook secret token. If not set, this field value is null; otherwise, please set a string value.",
+            default="",
+            blank="",
```

Comment:
Why need blank value for secret?

Reply:
BTW, we also support prompt input for the secret argument. https://azure.github.io/aaz-dev-tools/pages/usage/workspace-editor/#prompt-input-for-arguments